### DISCUSSION TITLE
Welcome to BPX Discussions!

### DISCUSSION THREAD

**Discussion by rtimms** [FaradayInstitution/BPX#21] [Announcements]
## 👋 Welcome to BPX Discussions!
We’re excited to have you here! BPX Discussions is a space where members of our community can connect, collaborate, and share insights. Here's what we encourage you to do:

- **Ask questions**: Curious about something? Don't hesitate to ask.
- **Share ideas**: Have a suggestion or innovative thought? We’d love to hear it.
- **Engage with others**: Join the conversation and connect with fellow community members.
- **Be welcoming and open-minded**: Respect different perspectives and foster a positive environment.

### 🚀 Get Started
Introduce yourself in the comments below! We'd love to hear about your work or research involving battery modelling and how you're currently using BPX (or planning to).

### 💡 Share Your Feedback
Discussions are the perfect place to voice your thoughts on the BPX roadmap.

- What features or improvements would make BPX more useful for you?
- What challenges or shortcomings have you encountered?
- Your input helps us make BPX better for everyone.

### 🔐 A Few Quick Reminders

- This is a public forum, so please don’t share sensitive information.
- If you’d like to contact us privately, feel free to email us at [bpx@faraday.ac.uk](mailto:bpx@faraday.ac.uk).

### 👥 Community Guidelines
We’re committed to maintaining a respectful and inclusive environment. By participating, you agree to follow our [Code of Conduct](https://github.com/FaradayInstitution/BPX/blob/main/CODE-OF-CONDUCT.md).

Thank you for joining the BPX community.

**Comment by rtimms** [comment]
Hi, I'm Robert Timms, Co-Founder at [Ionworks](https://ion-works.com/) and a core developer of the open-source battery modelling package [PyBaMM](http://www.pybamm.org/). At Ionworks we are building the modern software stack for batteries, powered by PyBaMM. 

Before founding Ionworks I was a Postdoctoral Research Assistant at the [Mathematical Institute, University of Oxford](https://www.maths.ox.ac.uk/). My research focused on the Mathematical Modelling of Batteries and was conducted within the Multi-Scale Modelling Fast Start Project of the [Faraday Institution](https://faraday.ac.uk/).

I have been leading PyBaMM’s contribution to the [Battery Parameter eXchange](https://bpxstandard.com/), an initiative led by the Faraday Institution which aims to bring the benefits of physics-based battery models to companies large and small, enabling information flow across the battery industry, reducing cost and time to market and stimulating innovation. 

You can get in touch via email [robertwtimms@gmail.com](mailto:robertwtimms@gmail.com), twitter [@robertwtimms](https://twitter.com/robertwtimms), or on the PyBaMM slack. For more information, you can visit my [website](https://www.robertwtimms.com/).

**Comment by jamiemfoster** [comment]
Hello,
I'm Jamie Foster, Reader in Industrial and Applied Mathematics at the University of Portsmouth. I have around a decade's worth of experience in modelling lithium-ion batteries and a co-leader of the dandeliion project (dandeliion.com); a software for lithium-ion battery simulation. Feel free to email at jamie.foster@port.ac.uk.
Cheers!
